extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -384,13 +390,14 @@ export class SubscriptionService {
 
   /**
    * Unblock document based on grant state and selected platform
+   * @param {boolean=} doPlatformSelection
    * @private
    */
-  startAuthorizationFlow_() {
+  startAuthorizationFlow_(doPlatformSelection = true) {
     this.platformStore_.getGrantStatus()
         .then(grantState => {this.processGrantState_(grantState);});
 
-    this.selectAndActivatePlatform_();
+    doPlatformSelection && this.selectAndActivatePlatform_();
```

Comment:
Switch to `if () {}` per AMP's styleguide.

Reply:
Done